gcapi/client.py

```diff
@@ -723,20 +723,6 @@ def update_archive_item(
                 "lung-volume": 1.9,
             },
         )
-        If you provide a value or file for an existing interface of the archive
-        item, the old value will be overwritten by the new one, hence allowing you
-        to update existing archive item values.
-        For images that are already associated with an archive item, you can
-        also change the interface type (e.g. from generic medical image to
-        generic overlay) by providing the link to the existing image together
-        with the new interface slug you would like to use:
-        client.update_archive_item(
-            archive_item_pk=items[0]['id'],
-            values={
-                "generic-overlay":
-                "https://grand-challenge.org/api/v1/cases/images/.../",
-            }
-        )
```

Comment:
Why was this help text removed?

Reply:
This functionality has been removed in gc, see https://github.com/comic/grand-challenge.org/pull/3156.

Reply:
Shouldn't that then be replaced with what was proposed: e.g. removing the image then adding the new one? What happens when a user tries to do this now?

Reply:
I could not find if that replacement functionality was actually implemented. What happens now (in the test at least) is that a new CIV is created with the new interface instead of replacing the old one.

I see now that I actually removed a bit too much here. The first sentence should not have been removed, adding it back now.